tool/create-exercise

```diff
@@ -226,38 +242,32 @@ Future main(args) async {
   await new File("${exerciseDir.path}/pubspec.yaml").writeAsString(pubTemplate(name));
 
   if (arguments["spec-path"] != null) {
-    // The output from test generation is not always well-formatted, use dartfmt to clean it up
-    final res = await Process.run("dartfmt", ["-l", "120", "-w", testFileName], runInShell: true);
-    if (!res.stdout.toString().isEmpty) {
-      stdout.write(res.stdout);
-    }
-
-    if (!res.stderr.toString().isEmpty) {
-      stderr.write(res.stderr);
-    }
-
-    if (res.exitCode != 0) {
-      stderr.write("Warning: dartfmt exited with a code of ${res.exitCode}, '$testFileName' is likely malformed.\n");
+    // Generate README
+    final dartRoot = "${dirname(Platform.script.toFilePath())}/..";
+    final configletLoc = "$dartRoot/bin/configlet";
+    final genSuccess = await runProcess(
+        configletLoc, ["generate", "$dartRoot", "--spec-path", arguments["spec-path"], "--only", name]);
+    if (genSuccess) {
+      stdout.write("Successfully created README.md\n");
     } else {
-      stdout.write("Successfully created a rough-draft of tests at '$testFileName'.\n");
-      stdout.write("You should check this over and fix or refine as necessary.\n");
+      stderr.write("Warning: `configlet generate` exited with an error, 'README.md' is likely malformed.\n");
     }
   }
 
-  // Install deps
-  Directory.current = exerciseDir;
-
-  final res = await Process.run("pub", ["get"], runInShell: true);
-
-  if (!res.stdout.toString().isEmpty) {
-    stdout.write(res.stdout);
+  // The output from file generation is not always well-formatted, use dartfmt to clean it up
+  final fmtSuccess = await runProcess("dartfmt", ["-l", "120", "-w", exerciseDir.path]);
+  if (fmtSuccess) {
+	  stdout.write("Successfully created a rough-draft of tests at '$testFileName'.\n");
+	  stdout.write("You should check this over and fix or refine as necessary.\n");
+  } else {
+	  stderr.write("Warning: dartfmt exited with an error, files in '${exerciseDir.path}' may be malformed.\n");
   }
 
-  if (!res.stderr.toString().isEmpty) {
-    stderr.write(res.stderr);
-  }
+  // Install deps
+  Directory.current = exerciseDir;
 
-  assert(res.exitCode == 0);
+  final pubSuccess = await runProcess("pub", ["get"]);
+  assert(pubSuccess);
```

Comment:
I believe these changes were made in a separate PR that was already merged into master, no need to include them here.

I recommend doing a rebase onto master to remove any changes introduced by previous commits to master.

Reply:
Also  you don't necessarily need to update your local branch after making a PR, you can continue making commits to it and when the PR is ready to merge it can be update.

Reply:
Sorry guys, I messed up the commit history here. Rebasing is messing up everything, I'll be more careful next time.

Reply:
@raphalvessa No worries, if you want, I can rebase the branch for you.

Reply:
@Stargator Yes please, I don't seem to be able to get out of this mess. Thanks!

Reply:
@raphalvessa Done.